frontend/main.js

```diff
@@ -214,6 +214,9 @@ async function startApp () {
     throw e
   })
 
+  // Clear the temporary files cache on app startup
+  sbp('gi.db/filesCache/temporary/clear')
```

Comment:
Isn't this perhaps missing a `.catch()`?

Reply:
Update made accordingly in [this prev commit](https://github.com/okTurtles/group-income/pull/2941/commits/0f483cef20ff6070d5f5038a6115730ce620ba59)